manifests/d/Decred/Decrediton/1.7.7/Decred.Decrediton.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Decred.Decrediton
+PackageVersion: 1.7.7
+PackageLocale: en-US
+Publisher: Decred
+PackageName: Decrediton
+License: ISC License
+Copyright: Copyright © 2023 The Decred developers

```

```suggestion
License: ISC License
LicenseUrl: https://github.com/decred/decrediton/blob/master/LICENSE
Copyright: Copyright (c) 2016-2022 The Decred developers
CopyrightUrl: https://github.com/decred/decrediton/blob/master/LICENSE
```
Additional metadata, also changed copyright to whatever is written at the CopyrightUrl